docs: update readme with monitoring and redshift commands (#2)
braze-inc/growth-shares-lambda-events-purchases-import: README.md

<kbd><img src="./images/trigger.png"></kbd>

:tada: You're ready to start sending custom events and purchases to Braze!

### Monitoring and Troubleshooting

> The function logs helpful messages while it is executing to help you understand if is processing the file correctly. The logs are available in the CloudWatch service. Under the **Monitor** tab, click on `View logs in CloudWatch`, and select the appropriate function log stream.

Reviewer: During execution, the function will log messages to help you understand if the file is being processed correctly. These logs are available on the CloudWatch service.

> The function logs helpful messages while it is executing to help you understand if is processing the file correctly. The logs are available in the CloudWatch service. Under the **Monitor** tab, click on `View logs in CloudWatch`, and select the appropriate function log stream.

Reviewer: ```suggestion
During execution, the function will log messages to help you understand if the file is being processed correctly. These logs are available on the CloudWatch service. Under the **Monitor** tab, click on `View logs in CloudWatch`, and select the appropriate function log stream.
```

In case of an unexpected error or in case of any questions, please [create an issue](https://github.com/braze-inc/growth-shares-lambda-events-purchases-import/issues) and try to include a example of the file that failed.

### Unloading from Redshift

> Redshift supports unloading relational data into JSON-like file. This example `unload` query exports events with properties to an S3 bucket.

Reviewer: "into a JSON-like file"

> Redshift supports unloading relational data into JSON-like file. This example `unload` query exports events with properties to an S3 bucket.

Reviewer: ```suggestion
Redshift supports unloading relational data into a JSON-like file. This example `unload` query exports events with properties to an S3 bucket.
```
